assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json

```diff
@@ -5,15 +5,17 @@
       },
       "description": "Policy to check required tags",
       "displayName": "HMCTS Tagging - CFT SBOX (DCD-CFTAPPS-SBOX)",
-      "enforcementMode": "Default",
+      "enforcementMode": "DoNotEnforce",
```

Comment:
Is this needed?

Reply:
Ok, good pint will switch back to Default.